### DISCUSSION TITLE
How to publish pages which aren't in the repository?

### DISCUSSION THREAD

**Discussion by ianthedev** [community/community#22254] [Other Feature Feedback, Questions, & Ideas]
<p><a href="https://github.com/kenwheeler/slick" rel="nofollow noopener">This project</a> is using a <a href="http://kenwheeler.github.io/slick/" rel="nofollow noopener">demo page</a>, which apparently does not exist in the repository.</p>
<p>How do I publish pages which aren’t in the repository like that?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/ianthedev">@ianthedev</a>,</p>
<p>Thanks for being here! At first glace, it appears the owner of that repo has this demo page’s code on a private repository. </p>
<p>GitHub Pages sites can only be served from private repositories on the Pro plan. GitHub Pages sites are still free on the Free plan for sites hosted from public repositories.</p>
<p>You can compare all the features for the various plans here:</p>
<p><a href="https://github.com/pricing" rel="nofollow noopener">https://github.com/pricing</a></p>
<p>I hope this helps!</p>

**Comment by ianthedev** [comment]
<p><a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> </p>
<p>Thank you for explaining! I see.</p>
